src/rain-report/rain-report.controller.spec.ts

```diff
@@ -35,4 +36,18 @@ describe('RainReportController', () => {
       expect(response).toStrictEqual(SEED_DATA);
     });
   });
+
```

Comment:
Tests to ensure that the `create()` controller method calls the underlying `RainReportService.create()` method with the expected values.